Dockerfile

```diff
@@ -270,4 +396,10 @@ ENV PYTHONUNBUFFERED=yessir
 # in ephemeral containers creating .pyc files after build is a waste of time
 ENV PYTHONDONTWRITEBYTECODE=aye
 
+# for devcontainer
+RUN useradd --create-home --shell=/bin/bash gisuser
+# should be UID:GID == 1000:1000
+RUN echo 'gisuser ALL=(ALL) NOPASSWD:ALL' >/etc/sudoers.d/gisuser
+USER gisuser
```

Comment:
I'm trying to use this base image in R2's AiHeatCleaner and I'm hitting an issue where `gunicorn` isn't found even though it is in the requirements.txt, I suspect because of this change. If I shell into the container I see:
`WARNING: The script gunicorn is installed in '/home/gisuser/.local/bin' which is not on PATH.`

Is the best way to handle this adding this line to my dockerfile you think?
`RUN export PATH="/home/gisuser/.local/bin:$PATH"`

Reply:
That actually didn't seem to work, the best way I've found is like so:
```
# switch to root to install dependencies
USER root

RUN pip install -r requirements.txt

# switch back to gisuser
USER gisuser

CMD gunicorn --bind :$PORT --workers 1 --threads 8 --timeout 0 main:app
```

Reply:
I saw this too, and have a fix someone on an uncommited branch. hope to have it committed once I complete testing of gdal 3.9.2 with PDFium...

IIRC, you can try `sudo pip install ---requirements` so it installs system-wide, not just as gisuser.

Reply:
Oh yeah that makes sense, I'll try that

Reply:
They seem six vs half-dozen. :-)

Reply:
Yeah, not sure why chat GPT didn't suggest that, seems simplest... 🤷